### ISSUE TITLE
Accessibility - All Incidents Screen - VoiceOver Not Starting on First Menu Item When Menu is Opened

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#10]
Once a person opens the “Menu” using VoiceOver, the next item it should read is the first thing in the menu instead of the rest of the header information in the application. If a user opens the menu, they should be focused on the menu so those using VoiceOver or a keyboard can read or tab to the first item. 

I would also expect the Menu to close if the user reads or tabs past the last item in the menu otherwise for the visual users with a keyboard wouldn't be able to see the content hidden behind it. Fixing this part might be a secondary thing in fixing the first item.

**Comment by aanthonyDOL** [comment]
Added Focus on First Menu Item for VoiceOver when expanded. The Menu closes on click per design.

**Comment by jfc3-dol** [comment]
@aanthonyDOL,
VoiceOver is not reading information behind the drop-down menu items, but it still reads the footer information. I would expect VoiceOver only to read the "Menu" items until the user closes the menu.
